extension/llm/export/builder.py

```diff
@@ -167,6 +176,59 @@ def capture_pre_autograd_graph(self) -> "LLMEdgeManager":
             )
         return self
 
+
+    def pt2e_calibrate(
+        self,
+        prepared_module,
+        calibration_tasks,
+        calibration_limit,
+        calibration_seq_length,
+        tokenizer_path,
+    ):
+        logging.info("Run calibration...")
+        try:
+            from executorch.examples.models.llama2.evaluate import EagerEvalWrapper, evaluate_model
+        except ImportError:
+            raise ImportError(
+                "Please install the llm eval dependency via examples/models/llama2/install_requirements.sh"
+            )
+
+        tokenizer = get_tokenizer(tokenizer_path)
+
+        def calibrate_template(module: torch.fx.GraphModule, tokenizer, string: str = "Once upon a time", max_len: int = 128):
+                # TODO: change criteria & support batch inputs if necessary
+                pos = torch.tensor(0, dtype=torch.int64)
+                token_list = [tokenizer.bos_id] + tokenizer.encode(string, bos=True, eos=False)
```

Comment:
@shewu-quic what did you use for `bos` and `eos`?

Reply:
bos and eos in TikTokenizer 
<img width="519" alt="image" src="https://github.com/user-attachments/assets/352e5608-a2e9-4026-a4dc-90707e674fbe">

Reply:
Oh I meant this part `tokenizer.encode(string, bos=True, eos=False)`, Did you enable `bos` and `eos`?

Reply:
Ohh~ is the same as your setting

Reply:
@shewu-quic does this PR work for you? If yes, can you kindly approve?

Reply:
`tokenizer.encode(string, bos=True, eos=False)` already pre append `bos`. Why do you need pre append an extra `tokenizer.bos_id`?

Reply:
Sorry to confuse you. Yes, we only one bos_id at the front. 
I think we could change to the following
```
token_list = tokenizer.encode(string, bos=True, eos=False)
```